docs/source/contributing.rst

```diff
@@ -0,0 +1,70 @@
+Undebt_: Contributing
+==========================
+
+.. _Undebt: index.html
+.. default-role:: code
+
+Getting Started
+---------------
+
+Undebt's development is taking place on Github, so please go ahead and `fork`_ the repository if you want to begin contributing.
+
+.. _`fork`: https://github.com/Yelp/undebt#fork-destination-box
+
+You'll then want to get a local copy of the code base:
+
+.. code-block:: bash
+
+    git clone git@github.com:<your-username>/undebt.git
+
+Getting Setup
+-------------
+
+It is highly recommended that you create a `virtual environment`_ before installing the project dependencies.
+
+.. _`virtual environment`: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+
+Both can be achieved by running:
```

Comment:
I think "both" == "creating virtualenv" + "install project deps"

But I agree this could probably be worded better.